src/hooks/Wallet/useJBWallet.ts

```diff
@@ -0,0 +1,48 @@
+import { readNetwork } from 'constants/networks'
+import { WalletContext } from 'contexts/Wallet/WalletContext'
+import { signOut, useSession } from 'next-auth/react'
+import { useContext } from 'react'
+import { useWallet } from './useWallet'
+
+export function useJBWallet() {
```

Comment:
new wrapper around `useWallet()` that includes EOA wallet connection as well as new keyp wallet